### ISSUE TITLE
tr_findTransfMatlab_mine_GS vs tr_findTransfMatlab_mine

### ISSUE THREAD

**Issue by PaoloGB** [GiovanniSena/LSFM#17]
The file 
tr_findTransfMatlab_mine_GS.m
contains a slightly different version of the original function.
However, when editing it you left the function name unchanged (i.e. both files contained function tr_findTransfMatlab_mine).
This is potentially risky because it is hard to know which version Matlab will actually use.
In fact, despite the change, I think Matlab was still using my version because there was a minor error in yours that would have prevented it from running.

Long story short: I fixed the minor error, tidied up both files and renamed your function correctly.

IMPORTANT: if you want to use the _GS version, you will have to call that in the start_call file instead of mine.
I will submit the tidied up code in a few minutes.

**Comment by PaoloGB** [comment]
In fact, the same issue occurs for
- tr_resizeStack_GS
- tr_takeSlices (this has 3 different versions, but I think the code would still use mine... So perhaps it is a good idea to check them)

**Comment by PaoloGB** [comment]
Ok, to summarize:

- **tr_findTransfMatlab_mine_GS**
- **tr_takeSlices_RSR**
- **tr_takeSlices_RSR_GS**
- **GUI_displayMIP_GS**
- **tr_resizeStack_GS**

these files were all renamed but still had the all function name within them. This could cause issues in Matlab because it is not clear what function would be executed.
I have changed the function names to match the file names but I have not changed the code that calls those functions.
So if you want to use one of the new versions you must find the code where they are called and replace the call with the correct name.

As an example, in _start_call_, line 159, replace:

_oldStack_small= tr_resizeStack(oldStack_trim, xy_scalefactor, 1);_

with

_oldStack_small= tr_resizeStack_GS(oldStack_trim, xy_scalefactor, 1);_

Since I have to way to check that the new functions work correctly, I will leave this to you and Todd.

I will leave this issue open.

**Comment by GiovanniSena** [comment]
Thank you, Paolo, it all sounds clear now. 

 @tfallese **Todd**, could you please take a look at those and keep only the versions we currently use? The differences are small, and the proliferation of variants just add to the confusion. Just bring everything down to the version you are using now, and then we will simply tinker in GitHub if necessary.
